### DISCUSSION TITLE
Warning: FYI, running unbundled (dev) frontend

### DISCUSSION THREAD

**Discussion by gianmariomanca** [fonsp/Pluto.jl#1704] [Questions about using Pluto] [answered]
After updating to 17.2 I get this warning when I start, anything to be concerned about?

`
┌ Warning: FYI, running unbundled (dev) frontend

└ @ Pluto ~/.julia/packages/Pluto/7TMtD/src/webserver/WebServer.jl:264
`

**Comment by chrisoffner3d** [comment]
I get the same warning, as well as **[this one](https://github.com/fonsp/Pluto.jl/discussions/1710)**.

**Comment by fonsp** [comment, marked as answer]
Nothing to worry about, the warning was removed in 0.17.3 :)
